src/gui/touchscreengui.cpp

```diff
@@ -1043,7 +1043,7 @@ void TouchScreenGUI::applyJoystickStatus()
 		if (i == 4 && !m_joystick_triggers_special1)
 			continue;
 
-		SEvent translated;
+		SEvent translated{};
```

Comment:
😂

Reply:
Not sure why this doesn't just have an empty diff?